ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -355,6 +353,13 @@ protected void doInBackgroundGuarded(Void... params) {
     }
   }
 
+  private static String getExtension(String path) {
+    if (path.contains(".")) {
+      return path.substring(path.lastIndexOf(".") + 1);
```

Comment:
Do we need to check for hashtag as well in this implementation?

Reply:
This would not require a check because the extension.. at least as i'm aware, shouldn't have a hashtag.